src/Agda2Hs/Compile/Term.hs

```diff
@@ -180,6 +179,8 @@ lambdaCase q es = setCurrentRange (nameBindingSite $ qnameName q) $ do
     -- simply return the body.
     [Hs.Match _ _ [] (Hs.UnGuardedRhs _ rhs) _] -> return rhs
     _ -> do
+      unlessM (fromMaybe False . fmap (`elem` [CaseOf, MonadicBind]) <$> asks isCompiling) $
```

Comment:
Could you use `isCompilingCaseOrBind` to make it more legible?

Reply:
Oops, leftover.